packages/devops/configs/servers.conf

```
@@ -455,6 +455,37 @@ server {
 
 }
 
+server {
+
+  listen       80;
+  listen       [::]:80;
+  server_name  ~^.*datatrak\.tupaia\.org$;
+  root         /home/ubuntu/tupaia/packages/datatrak-web/build;
```

```suggestion
  root         /home/ubuntu/tupaia/packages/datatrak-web/dist;
```
I wonder if this is why the build failed? Hopefully 🤞